crates/evm-state-queries/src/hyperlane/mod.rs

```diff
@@ -23,19 +23,25 @@ mod tests {
         if a new block was posted in the meantime.
     */
     async fn test_run_indexer() {
+        dotenvy::dotenv().ok();
```

Comment:
quick question, whats the diff between dotenv and dotenvy?

We depend on dotenv in some of the other crates, is there any advantage to dotenvy? I would recommend we stay consistent to avoid growing the dep graph accidentally

Reply:
Thanks, that's good to know!